mon-pix/app/components/challenge-item-generic.js

```diff
@@ -48,8 +48,17 @@ const ChallengeItemGeneric = Component.extend({
     return _.isInteger(this.challenge.timer);
   }),
 
-  displayFeedbackPanel: computed('_isUserAwareThatChallengeIsTimed', 'hasChallengeTimer', function() {
-    return !this.hasChallengeTimer || (this.hasChallengeTimer && this._isUserAwareThatChallengeIsTimed);
+  displayFeedbackPanel: computed('_isUserAwareThatChallengeIsTimed', 'hasChallengeTimer', 'displayWarningPage', function() {
+    return !this.hasChallengeTimer
+      || (this.hasChallengeTimer && this._isUserAwareThatChallengeIsTimed)
+      || !this.displayWarningPage;
+  }),
+
+  displayWarningPage: computed('answer', 'hasChallengeTimer', function() {
+    if (!this.answer && this.hasChallengeTimer) {
+      return true;
+    }
+    return false;
```

Comment:
Petite suggestion de simplification

```suggestion
    return !this.answer && this.hasChallengeTimer
```

Reply:
Bonne remarque ! À noter cependant qu'une version sécurisée est ici nécessaire puisque `answer` n'est pas un boolean mais un objet
Ça ressemblerait à ça : 
```js
return !!(!this.answer && this.hasChallengeTimer);
```

Le `!!` nous a paru moins clair

Reply:
Oui mais `!this.answer` convertit l'objet en boolean. Donc pas besoin du `!!`.

Reply:
Je suis d'accord mais il me semble qu'on a un historique avec cette conversion. C'était un peu bancal avec Ember + InlliJ préconise le `!!`. Je garde ça en tête pour la glimmerization où je suis plus confiant avec la conversion